harness/determined/common/api/request.py

```diff
@@ -142,7 +142,12 @@ def do_request(
 
     def _get_message(r: requests.models.Response) -> str:
```

Comment:
`_get_error_message` or `_get_error_str` would probably be more appropriate name for this func